auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -140,21 +140,30 @@
         <template #item-slot-Status="{ item }">
           <span>{{ status(item) }}</span>
           <!-- Future Effective Icon for PAID_FE status -->
-          <template v-if="status(item) === BusinessState.PAID_FE">
-            <v-tooltip bottom>
-              <template #activator="{ on, attrs }">
-                <v-icon
-                  class="ml-1"
-                  color="#F8661A"
-                  v-bind="attrs"
-                  v-on="on"
-                >
-                  mdi-information-outline
-                </v-icon>
-              </template>
-              <span>This filing will become effective on {{ formatDate(item.effectiveDate) }}</span>
-            </v-tooltip>
-          </template>
+          <IconTooltip
+            v-if="status(item) === BusinessState.PAID_FE"
+            icon="mdi-information-outline"
+            maxWidth="300px"
+            :iconStyling="{'font-size': '1.5em', 'margin-left': '4px'}"
+            :location="{top: true}"
+          >
+            <span>This filing will become effective on {{ formatDate(item.effectiveDate) }}</span>
+          </IconTooltip>
+
+          <!-- Alert Icon for CHANGE_REQUESTED status -->
+          <IconTooltip
+            v-if="status(item) === BusinessState.CHANGE_REQUESTED"
+            icon="mdi-alert"
+            maxWidth="300px"
+            :colour="'#F8661A'"
```

Comment:
Since the colour value is not a variable, you don't need to "bind" it. Try leaving off the `:` and removing the inside quotes.

Reply:
Thanks, updated